project/Dependencies.scala

```diff
@@ -52,6 +52,7 @@ object Dependencies {
   val rome = "rome" % "rome" % romeVersion
   val romeModules = "org.rometools" % "rome-modules" % romeVersion
   val scalaCheck = "org.scalacheck" %% "scalacheck" % "1.16.0" % Test
+  val scalaCollectionPlus = "com.madgag" %% "scala-collection-plus" % "0.11"
```

Comment:
Who’s this madgag fellow?

Reply:
I don't know but I have often wondered myself 😆

Reply:
I purchased [madgag.com](https://web.archive.org/web/20050128112700/http://madgag.com/) back in 2003 (as the domain for a comedy listings site I was trying to create as a side-project - I didn't realise there was already the [_Chortle_](https://en.wikipedia.org/wiki/Chortle) website doing the same thing!). As the website was for getting to comedy gigs in obscure places, I wanted it to be accessible by mobile phone, which at the time was at the very edge of what phones could do - the iPhone wouldn't launch for another 4 years. Phones had screens, but not touchscreens, just numeric keypads, like this:

![image](https://user-images.githubusercontent.com/52038/188638999-209aed9c-dbb7-4e0a-8822-011377439e31.png)

At the time the fastest way of typing text - eg a url - into a phone was using [T9 (predictive text)](https://en.wikipedia.org/wiki/T9_(predictive_text)), so I wanted my domain name to be not only comedy-related, but easy to type in - that meant finding a name that _only_ used the first letters of the numeric keypad (`A`, `D`, `G`, `J`, `M`, `P`, `T`, `W`). I came up with `madgag`, which is very easy to type using a T9 keypad...!

Although I had a fun time writing the code for madgag.com, it was never fully developed, and I turned it off a few years later. However, in writing the site, I happened to learn a bunch of stuff about databases and object-relational-mapping, which - guess what - came up when I had my phone interview at the Guardian! So the interview went well, and helped me get the job here!